<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamWindow</class>
 <widget class="QDialog" name="TeamWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="teamDetailsBackButton">
     <property name="text">
      <string>Back to Team Details</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="teamInfoVerticalLayout">
     <item>
      <layout class="QFormLayout" name="teamInfoFormLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="teamInfoNameLabel">
         <property name="text">
          <string>Team Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="teamInfoNameInput"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="teamInfoOffensiveRatingLabel_2">
         <property name="text">
          <string>Offensive Rating: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="teamInfoOffensiveRatingInput"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="teamInfoOffensiveRatingLabel">
         <property name="text">
          <string>Defensive Rating: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="teamInfoDefensiveRatingInput"/>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="buttonHorizontalLayout">
         <item>
          <widget class="QPushButton" name="deleteTeamButton">
           <property name="text">
            <string>Delete Team</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateTeamButton">
           <property name="text">
            <string>Update Team</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
